<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGetCuStrapPosWidget</class>
 <widget class="QWidget" name="CGetCuStrapPosWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="lay_widget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>连接片方向</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_direction">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>上下</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>左右</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>小阈值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_minthre">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Cu位置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_cuPos">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>上</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>下</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>左</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>右</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_getCuPos">
        <property name="text">
         <string>检测</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>阈值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_thre">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>220</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_pamRead">
        <property name="text">
         <string>获取参数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_detect">
        <property name="text">
         <string>检测</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spin_thre</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_detect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
